docs/johannes_mueller/yolo_from_omero/train_yolo.ipynb

```diff
@@ -0,0 +1,705 @@
+   "source": [
+    "image_to_upload = io.imread(test_image)\n",
+    "image_id = ezomero.post_image(\n",
+    "    conn, image=image_to_upload[None, None, None, :], dim_order='tczyx',\n",
+    "    dataset_id=260, image_name='example_prediction'\n",
+    "    )\n",
+    "\n",
+    "roid_id = ezomero.post_roi(conn, image_id=image_id, shapes=rectangles)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we check on our OMERO server we see our predictions:\n",
+    "\n",
+    "![OMERO predictions](./imgs/image8.PNG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Final thoughts\n",
+    "\n",
+    "That's it! This is quite a lengthy tutorial, so props if you made it all the way here. \n",
+    "\n",
+    "### Used data\n",
+    "\n",
+    "All the data used in this tutorial is [publically available](https://zenodo.org/records/13329153) at the courtesy of Louis Hewitt ([Grapin-Botton Lab](https://www.mpi-cbg.de/research/researchgroups/currentgroups/anne-grapin-botton/research-focus), MPI-CBG, Dresden, Germany). Without their collaboration, you wouldn't be able to look at this tutorial. So, thank you!\n",
+    "\n",
+    "The data of this tutorial is available on Zenodo as a packed up OMERO project. You can use it by downloading the stored jar file and using the [omero-cli-transfer](https://github.com/ome/omero-cli-transfer) package to upload it to your OMERO server and then walk yourself through this tutorial from the start. To do so, log into omero from the command line like this:\n",
+    "\n",
+    "```bash\n",
+    "omero login -s <your_omero_server> -u <your_username> -g <your_group>\n",
+    "```\n",
+    "\n",
+    "Then you can upload the jar file to your server:\n",
+    "\n",
+    "```bash\n",
+    "omero transfer unpack <path_to_jar_file>\n",
+    "```\n",
+    "\n",
+    "### Licenses\n",
+    "\n",
+    "You have seen that the syntax to set up the training for the yolo model is exceptionally easy, which is a huge strongpoint for yolo - not to mention that the model is quite fast and accurate. A grain of salt for using yolo is its AGPL license, which introduces quite strict requirements if you use it in your project. If you are planning to use yolo in a commercial project, you need to contact the developers for a proper license. For a more comprehensive overview of the licenses, check out [Robert Haase's slides on the subject](https://f1000research.com/slides/10-519).\n",
+    "\n",
+    "### How much training data is enough?\n",
+    "\n",
+    "That's a question that is hard to answer. According to the [yolo docs on the subject](https://docs.ultralytics.com/guides/data-collection-and-annotation/#how-many-images-do-i-need-for-training-ultralytics-yolo-models), some 100 annotated images and training for 100 epochs can be enough to get started - consider annotating (order of magnitude) 1000s of images *per class* for a more robust model.\n",
```

Comment:
You could add a link to this section in the Annotation part.

Reply:
Added in 9d683d997f89822c4eb2818b63187ec8bef2f0e6